src/database/shell.c

```diff
@@ -23080,7 +23080,7 @@ int SQLITE_CDECL wmain(int argc, wchar_t **wargv){
   char **argv;
 #endif
 #ifdef SQLITE_DEBUG
-  sqlite3_uint64 mem_main_enter = sqlite3_memory_used();
+  sqlite3_int64 mem_main_enter = sqlite3_memory_used();
```

Comment:
Is the type change because of the version change or unrelated?

Reply:
This is an Sqlite3 change. We only import the patchset.